packages/document/module-doc/docs/en/api/config/build-config.mdx

```diff
@@ -644,7 +644,7 @@ For more information about JSX Transform, you can refer to the following links:
 
 - [React Blog - Introducing the New JSX Transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).
 - [esbuild - JSX](https://esbuild.github.io/api/#jsx).
-  :::
+:::
```

Comment:
Usually we need to add an blank line before `:::` to make prettier working correctly